Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.0
```

Comment:
I think this change means that SPM users on Swift 4 will no longer be able to use SwiftGRPC. I don't think we should cross that bridge yet.

Reply:
@rebello95 instead you can leave this at 4 (or maybe it has to be 4.2 for this feature) and add explicit language versions https://github.com/apple/swift-protobuf/blob/3dd275f36c2089aca0446d75c55b7c325fb36cf2/Package.swift#L34

Then with the Swift 5 tools I _believe_ it will pass `-swift-version 5`

Reply:
Updated!